container/ip-address-expansion/nodejs/appWorker.js

```diff
@@ -0,0 +1,137 @@
+/*
+  Copyright (c) 2017, F5 Networks, Inc.
```

Comment:
We are not using same license on GUI code ? Is that okay ?

Reply:
added f5 copyright, and apache 2.0 license to presentation source files.